job-dsls/jobs/pr_jobs.groovy

```diff
@@ -116,7 +118,7 @@ def final REPO_CONFIGS = [
         ]
 ]
 
-def final SONARCLOUD_ENABLED_REPOSITORIES = ["optaplanner"]
+def final SONARCLOUD_ENABLED_REPOSITORIES = ["optaplanner", "drools"]
```

Comment:
don't we want to turn on this for jbpm too?

Reply:
We do, sure we do :) once https://github.com/kiegroup/jbpm/pull/1409 is merged and corresponding nightly job provides some initial data (and most importantly does not fail).

That's how I am enabling sonarcloud:
1. include profiles for code coverage into the repository
2. create a nightly job to make sure it works
3. enable it for PRs

After the initial work the steps 2. and 3. are trivial.

Reply:
ok, great!